ArticleTemplates/videoCaptionTemplate.html

```diff
@@ -0,0 +1,3 @@
+<figcaption>
+    <span class="element-image__caption"><span data-icon="&#xe043;" class="figure__caption__icon" aria-hidden="true"></span>__CAPTION__</span>
```

Comment:
@Leyths looks good, however there's one thing, could you change "element-image__caption" to "element-video__caption"